src/hearings/containers/request-hearing/hearing-judge/hearing-judge.component.html

```diff
@@ -43,7 +43,7 @@ <h1 class="govuk-fieldset__heading">{{ 'Name of the judge' | rpxTranslate }}</h1
               <input class="govuk-radios__input" id="noSpecificJudge" name="specificJudge"
                 (change)="showSpecificJudge('No');" formControlName="specificJudge" type="radio" value="No"
                 data-aria-controls="conditional-judge-type">
-              <label class="govuk-label govuk-radios__label" for="noSpecificJudge">{{ 'No' | rpxTranslate }}</label>
+              <label class="govuk-label govuk-radios__label" for="noSpecificJudge">{{ 'Do you want a specific judge?' | rpxTranslate : null : 'No' }}</label>
```

Comment:
Minor comment: 'Do you want a specific judge?' especially as it is repeated should be a string constant/variable